src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
 body {
   margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100vw;
+  height: 100vh;
 }
+
+* {
+  box-sizing: border-box;
+}
+
+@import 'main';
+@import 'variable';
```

Comment:
Ensure that the 'main' and 'variable' SCSS files exist in the same directory or provide the correct path if they are located elsewhere. This is necessary for the imports to work correctly.